packages/shared/src/components/squads/Details.tsx

```diff
@@ -70,32 +68,19 @@ export function SquadDetails({
   createMode = true,
   children,
 }: SquadDetailsProps): ReactElement {
-  const { user } = useAuthContext();
   const {
     name,
     handle,
     description,
     memberPostingRole: initialMemberPostingRole,
     memberInviteRole: initialMemberInviteRole,
   } = form;
-  const { data: status } = useQuery<SquadStatus>(
-    generateQueryKey(RequestKey.SquadStatus, user),
-    () => {
-      // TODO: MI-344 we need to add a condition if they are already an approved public squad that went private to return approved
-      if (form.public) {
-        return SquadStatus.Approved;
-      }
-
-      // TODO: MI-344 add another 2 if statement here to check if the squad is rejected/pending
-
-      return SquadStatus.InProgress;
-    },
-    {
-      ...disabledRefetch,
-      cacheTime: Infinity,
-      enabled: !isNullOrUndefined(form?.flags?.totalPosts) && !createMode,
-    },
-  );
+  const isRequestsEnabled =
+    !createMode && !form.public && !form.flags?.totalPosts && !!form?.id;
```

Comment:
we were using `!isNullOrUndefined` above, don't we need it here?

Reply:
Since u're integrating also MI-344 should the above be here?

`TODO: MI-344 add another 2 if statement here to check if the squad is rejected/pending`

Reply:
Shouldn't matter which one we would use. If the value is `0`, you don't have to make any request still since that would mean you have never requested.